test/server-unit/dates.spec.js

```diff
@@ -0,0 +1,68 @@
+const dates = require('../../server/lib/template/helpers/dates');
+const { expect } = require('chai');
+
+
+const newLocal = function () {
+
+  it('#date() should format a date is "DD/MM/YYYY" format.', () => {
+    let dat =  new Date('2015-03-25');
+    var expected = '25/03/2015';
+    var formated = dates.date(dat);
+    expect(formated).to.equal(expected);
+  });
+
+  it('#date() should return an empty string ("") if the date is null.', () => {
+    let dat =  null;
+    var expected = '';
+    var formated = dates.date(dat);
+    expect(formated).to.equal(expected);
+  });
+
+  it('#date() should allow you to specify a custom format.', () => {
+    let dat =  new Date('2015-03-25');
+    var expected = '03/2015';
+    let format =  'MM/YYYY';
+    var formated = dates.date(dat, format);
+    expect(formated).to.equal(expected);
+  });
+
+  it('#timestamp() should return an empty string ("") if the date is null.', () => {
+    let dat =  null;
+    var expected = '';
+    var formated = dates.timestamp(dat);
+    expect(formated).to.deep.equal(expected);
```

Comment:
For strings, you don't need `deep`.  The `deep` call will recursively descend into objects.